website/blog/2022-08-31-august-product-update.md

```diff
@@ -0,0 +1,64 @@
+
+You’ll hear more in [Tristan’s keynote](https://coalesce.getdbt.com/agenda/keynote-the-end-of-the-road-for-the-modern-data-stack-you-know), but this feels like a good time to remind you that Coalesce isn’t just for answering tough questions… it’s for surfacing them. For sharing challenges we’ve felt in silos, finding the people you want to solve them with, and spending the rest of the year chipping away at them. As Tristan says in his latest blog, [that’s how this industry moves forward](https://www.getdbt.com/blog/finding-our-next-big-problem/).
+
+[REGISTER NOW](https://coalesce.getdbt.com/register?utm_medium=email&utm_source=hs_email&utm_campaign=q3-2023_coalesce-2022_awareness&utm_content=connect_product-update_)
+
+<!--truncate-->
+
+## **What's New**
+
+- **dbt Core v1.3 beta:** Do you use Python for analytics? The first beta prerelease of dbt Core v1.3—including support for dbt models written in Python—is [ready to explore](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.3)! Check it out, and read more about dbt supported Python models [in our docs](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/python-models).
+- **Technology Partner Program:** We just launched our new [Technology Partner Program](https://www.getdbt.com/blog/dbt-labs-technology-partner-program/) with 40+ friends in the Modern Data Stack to provide consistent support for seamless integrations joint-users can trust. Check our new [dbt Cloud integrations page](http://www.getdbt.com/product/integrations) for what’s available today!
+- **Single-tenant users:** dbt Cloud v1.1.60 is now available on dbt Cloud Enterprise.
+
+## What’s Better
+
+- **dbt Cloud UI:** The [new dbt Cloud UI](https://www.getdbt.com/blog/the-dbt-cloud-ui-is-getting-a-makeover/) is in beta, and can be opted-into by any multi-tenant dbt Cloud customer. Cleaned-up interface, better ergonomics, fewer clicks to frequently-used screens.
+- **dbt Cloud IDE:** Did you catch [Staging last month](https://www.getdbt.com/blog/staging-highlights-the-latest-from-dbt-labs/) (our quarterly product update)? The dbt Cloud IDE has been overhauled for greater speed and performance, and is now in beta—[enroll to check it out!](https://bit.ly/dbt-cloud-ide-beta)
+
+## New Resources
+
+**Things to try** 🛠️
+
+- **dbt_artifacts v1.2.0:** [Brooklyn Data Co just shipped a pretty significant re-write to the dbt_artifacts package](https://brooklyndata.co/blog/dbt-artifacts-v100). Capture all the metadata generated by dbt at the end of an invocation (project nodes, success rate, test results, etc), and store directly in Snowflake, Databricks, or BigQuery for immediate analysis.
+- https://github.com/dbt-labs/dbt-jsonschema/: If you do any development in VS Code, this repo unlocks autocomplete and validation for dbt’s YAML files. Find those tests that never ran because you messed up the indentation. *Not that that would ever happen to you.*
```

Comment:
Can you create a bolded title for this bullet like you did for others instead of just linking to this to provide more context off the bat for what this is?